v2/constants.go

```diff
@@ -51,23 +51,28 @@ const (
 
 // Constants related to defined url path names and parameters in the v2 service APIs
 const (
-	All           = "all"
-	Id            = "id"
-	Pushed        = "pushed"
-	Count         = "count"
-	Device        = "device"
-	DeviceIdParam = "deviceId"
-	Start         = "start"
-	End           = "end"
-	Age           = "age"
-	Scrub         = "scrub"
-	Type          = "type"
-	Name          = "name"
-	Manufacturer  = "manufacturer"
-	Model         = "model"
-	Offset        = "offset" //query string to specify the number of items to skip before starting to collect the result set.
-	Limit         = "limit"  //query string to specify the numbers of items to return
-	Labels        = "labels" //query string to specify associated user-defined labels for querying a given object. More than one label may be specified via a comma-delimited list
+	All          = "all"
+	Id           = "id"
+	Created      = "created"
+	Modified     = "modified"
+	Pushed       = "pushed"
+	Count        = "count"
+	Device       = "device"
+	DeviceId     = "deviceId"
+	DeviceName   = "deviceName"
+	ProfileName  = "profileName"
+	ServiceName  = "serviceName"
+	Start        = "start"
+	End          = "end"
+	Age          = "age"
+	Scrub        = "scrub"
+	Type         = "type"
+	Name         = "name"
+	Manufacturer = "manufacturer"
+	Model        = "model"
+	Offset       = "offset" //query string to specify the number of items to skip before starting to collect the result set.
+	Limit        = "limit"  //query string to specify the numbers of items to return
+	Labels       = "labels" //query string to specify associated user-defined labels for querying a given object. More than one label may be specified via a comma-delimited list
```

Comment:
I still couldn't find `ValueType` as defined in the constant list.  Do you wanna check one more time?

Reply:
sorry, I missed a git step.  it is good now